### ISSUE TITLE
“验证集的结果好与提交的测试集结果很差”

### ISSUE THREAD

**Issue by xiaoxusanheyi** [Cc-Hy/CMKD#40]
针对kitti-raw的数据集，我们在修改了文中的loss后，按照github上的多GPU分阶段训练(second)执行训练验证后，结果很好，进行测试，把结果提交上去后结果很差很差，不知道为什么？过拟合了？

**Comment by Cc-Hy** [comment]
I think if you get extremely bad results, it's generally not because of overfitting, but because there are some significant errors in your model or your code.

**Comment by xiaoxusanheyi** [comment]
1.有重大错误的话，为什么我验证集的结果很好？
2.由于我是80的服务器，原文中batch_size的4，显存不够，于是就设置为了2，那么我是否需要把两个学习率lr（bev.yaml和V2.yaml）降为一半?或者说需要把epoch增加一倍再训练?

**Comment by Cc-Hy** [comment]
Hi,
1. 'significant errors' could come from many sources. The performance on the validation set is good indicates that your model should be correct, but you might have used the wrong data for training or the wrong data for inference.
2. I think the change to the learning rate is reasonable, but in my experience, the learning rate does not lead to the above results.

**Comment by xiaoxusanheyi** [comment]
ok，感谢作者的回复

**Comment by TimGor1997** [comment]
> 针对kitti-raw的数据集，我们在修改了文中的loss后，按照github上的多GPU分阶段训练(second)执行训练验证后，结果很好，进行测试，把结果提交上去后结果很差很差，不知道为什么？过拟合了？

大佬！我想请问你前面问题中提到的三个类别的mAP全是00是怎么解决的呢？
